forms-shared/src/generator/uiOptionsTypes.ts

```diff
@@ -158,7 +158,7 @@ export type ObjectFieldUiOptions = Pick<WidgetUiOptions, 'spaceTop' | 'spaceBott
 export type StepUiOptions = {
   stepQueryParam?: string
   stepperTitle?: string
-}
+} & Pick<ObjectFieldUiOptions, 'description' | 'descriptionMarkdown'>
```

Comment:
If i understand it correctly, `descriptionMarkdown` is just a boolean, in that case `isDescriptionMarkdown` is better i think

```suggestion
} & Pick<ObjectFieldUiOptions, 'description' | 'isDescriptionMarkdown'>
```

Reply:
Good point!

However I would have to change these all:
![image](https://github.com/user-attachments/assets/2e723eb0-427c-414d-989e-a4f62af957ff)

I've created a task https://github.com/bratislava/private-konto.bratislava.sk/issues/824